src/components/filters/MeasureValueFilter/DropdownBody.tsx

```diff
@@ -36,21 +39,28 @@ class DropdownBodyWrapped extends React.PureComponent<IDropdownBodyProps, IDropd
     constructor(props: IDropdownBodyProps) {
         super(props);
 
-        const { operator, value } = props;
+        const { operator, value, usePercentage } = props;
 
         this.state = {
             operator: operator || Operator.ALL,
-            value: value || {},
+            value: (usePercentage ? this.convertToPercentageValue(value) : value) || {},
         };
     }
 
     public render() {
-        const { onCancel, intl } = this.props;
+        const { onCancel, warningMessage, intl } = this.props;
         const { operator } = this.state;
 
         return (
             <div className="gd-mvf-dropdown-body gd-dialog gd-dropdown overlay s-mvf-dropdown-body">
                 <div className="gd-mvf-dropdown-content">
+                    <div className="gd-mvf-dropdown-section">
+                        {warningMessage && (
```

Comment:
Shouldn't condition include `gd-mvf-dropdown-section` as well? This way some unwanted paddings can be rendered.